src/tabpfn/regressor.py

```diff
@@ -707,6 +707,34 @@ def predict(  # noqa: C901, PLR0912
 
         return logit_to_output(output_type=output_type)
 
+    def get_embeddings(self, X: XType) -> np.ndarray:
+        """
+        Get the embeddings for the input data `X`.
+
+        Parameters:
+            X (XType): The input data.
+        Returns:
+            np.ndarray: The computed embeddings for each fitted estimator.
+        """
+        check_is_fitted(self)
+
+        X = validate_X_predict(X, self)
+        X = _fix_dtypes(X, cat_indices=self.categorical_features_indices)
+        X = self.preprocessor_.transform(X)
+
+        embeddings: list[np.ndarray] = []
+
+        for output, config in self.executor_.iter_outputs(
```

Comment:
Perhaps I am missing something but I don't see how you are getting the train/test embeddings for X. Shouldn't the `iter_outputs` method be passed a new param that would invoke the model with `only_return_standard_out=False`?